Lil-Peep/src/styles/modules/site-footer.scss

```diff
@@ -0,0 +1,234 @@
+footer {
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  background-color: #fff;
+  margin-top: 15px;
+  border-top-style: solid;
+  border-top-color: rgba(28, 26, 26, 0.1);
+  border-top-width: 1px;
+}
+
+.footer-content {
+  margin: 0 40px;
+
+  .social-media-container {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    width: 100%;
+
+    .social-media-box {
```

Comment:
We can remove this nested class by moving it to the line # 15. Why? If the class `social-media-box` is not depending of the `social-media-container` class then we can safely move this class but keeping it inside the `.footer-content` section.

Reply:
Ohh! I thought it would be better if everything were sectioned just like how the structure is in HTML. Alright! I'm on it!